src/EFCore.Sqlite.Core/Query/Internal/SqliteMathTranslator.cs

```diff
@@ -78,7 +84,7 @@ public SqliteMathTranslator(ISqlExpressionFactory sqlExpressionFactory)
         {
             RelationalTypeMapping? typeMapping;
             List<SqlExpression>? newArguments = null;
-            if (sqlFunctionName == "max" || sqlFunctionName == "max")
+            if (sqlFunctionName == "max" || sqlFunctionName == "min")
```

Comment:
Oops - so we weren't inferring correctly for min previous...

```suggestion
            if (sqlFunctionName is "min" or "max")
```

Reply:
lol, I reject your new age sorcery. (I'm going to change it in the big Math PR anyway.)

> Oops - so we weren't inferring correctly for min previous...

Yeah, but we'd still infer it from the first argument which is most likely your column anyway. Also type-mappings are a lot less important on SQLite since you can just ask the ADO.NET provider for whatever type you want. So yeah, there was probably a corner-case involving value converters somewhere that this fixes.

Reply:
> lol, I reject your new age sorcery. (I'm going to change it in the big Math PR anyway.)

Oh that's nothing, take a look at what @maumar [has to put up with](https://github.com/dotnet/efcore/blob/main/src/EFCore.Relational/Query/RelationalQueryableMethodTranslatingExpressionVisitor.cs#L1825) :rofl: